DBM_TBC5Man/AQTBC.lua

```diff
@@ -0,0 +1,493 @@
+local tbcCthun = DBM:NewBossMod("AQTBC", DBM_CTHUN_T65_NAME, DBM_AQ40_DESCRIPTION, DBM_ZONE_AQ40, "TBC5ManTab", 7)
```

Comment:
```suggestion
local Cthun = DBM:NewBossMod("Cthun65", DBM_CTHUN_T65_NAME, DBM_AQ40_DESCRIPTION, DBM_ZONE_AQ40, "TBC5ManTab", 7)
```

Can be a shorter local variable name `Cthun`. Also, the identifier `Cthun65` is used internally in DBM communications.